editions/tw5.com/tiddlers/demonstrations/TableOfContentsMacro Tabbed Example.tid

```diff
@@ -1,13 +1,15 @@
-created: 20150221203607000
-modified: 20150221210000000
-title: Example Table of Contents: Tabbed Internal
 caption: Tabbed Internal
+created: 20150221203607000
+modified: 20190617113004175
 tags: table-of-contents-example
-order: 5
+title: Example Table of Contents: Tabbed Internal
+type: text/vnd.tiddlywiki
 
 <$macrocall
 	$name="toc-tabbed-internal-nav"
 	tag="TableOfContents"
+        sort="" 
```

Comment:
Is there a reason to add these blank attributes? Can we not rely on the defaults?

Reply:
Yes. ... That's example tiddlers, which should be copy pasted by users. It's much easier to remove attributes that you don't need, than create typos if you type them on your own.

Reply:
Good point.